### PR TITLE
default content type to audio/mpeg, prevent bad type from fixer

### PR DESCRIPTION
**Pull request by kookster**

### CODE REVIEW

File: app/models/media_resource.rb [hunk 1]
```diff
@@ -46,13 +46,23 @@ def update_from_fixer(fixer_task)
   end
 
   def update_attributes_with_fixer_info(info)
-    self.mime_type = info['content_type']
-    self.file_size = info['size'].to_i
+    # this is not working
+    # self.mime_type = info['content_type']
```
Comment (cavis) [inline app/models/media_resource.rb]: This isn't working in the "is null" sense, or in the "doesn't have a sane default" sense?

File: app/models/media_resource.rb [hunk 2]
```diff
@@ -46,13 +46,23 @@ def update_from_fixer(fixer_task)
...
     self.sample_rate = info['sample_rate'].to_i
     self.channels = info['channels'].to_i
     self.duration = info['length'].to_f
     self.bit_rate = info['bit_rate'].to_i
     self
   end
+
+  def update_mime_type_with_fixer_info(info)
+    if info['content_type'].nil? || info['content_type'] == 'application/octect-stream'
+      self.mime_type ||= 'audio/mpeg'
```
Comment (cavis) [inline app/models/media_resource.rb]: You're thinking this is a safe-enough assumption, just because we deal with so little video?

Comment (kookster) [inline app/models/media_resource.rb]: yes, that is my thought. If it is ever wrong, we can console update the enclosure to have a correct value

**Comment by cavis** [comment]
👍

**Pull request merged**